GitCommands/Git/GitRef.cs

```diff
@@ -12,19 +12,19 @@ public class GitRef : IGitRef
         private readonly string _remoteSettingName;
 
         /// <summary>"refs/tags/"</summary>
-        public static readonly string RefsTagsPrefix = "refs/tags/";
+        public const string RefsTagsPrefix = "refs/tags/";
```

Comment:
❕ I prefer keeping `static readonly` for non-private fields, except in cases where those values are used in constant contexts (e.g. attribute constructors). Using `const` changes the compilation behavior for external references in a way that loses the original reference, and operations like finding references via metadata are unnecessarily complicated.

This occurs in several places in this commit, but I've only commented here.

Reply:
Use static readonly - RussKie will look for these too.
(The dlls are not reused right now, but public const is considered bad practice.)

Reply:
I've backed out the changes for non-private (public and internal, in practice) fields.